src/test/java/com/devonfw/demoquarkus/service/v1/ProductRestServiceTest.java

```diff
@@ -23,75 +26,77 @@
   void getAll() {
 
-    Response response = given().when().contentType(MediaType.APPLICATION_JSON).get("/animals").then().statusCode(200)
+    Response response = given().when().contentType(MediaType.APPLICATION_JSON).get("/products").then().statusCode(200)
         // .body("$.size()", equalTo(2))
         .extract().response();
 
-    PageResultDTO<AnimalDto> animalsReturned = response.as(new TypeRef<PageResultDTO<AnimalDto>>() {
+    PageResultDTO<ProductDto> productsReturned = response.as(new TypeRef<PageResultDTO<ProductDto>>() {
     });
 
     // we import data from /import.sql - ergo expect 1 result
-    assertEquals(2, animalsReturned.getTotalElements());
+    assertEquals(2, productsReturned.getTotalElements());
   }
 
   @Test
   void getNonExistingTest() {
 
-    Response response = given().when().contentType(MediaType.APPLICATION_JSON).get("/animals/doesnoexist").then().log()
+    Response response = given().when().contentType(MediaType.APPLICATION_JSON).get("/products/doesnoexist").then().log()
         .all().statusCode(404).extract().response();
   }
 
   @Test
   @WithDBData(value = "data/empty.xls", deleteBeforeInsert = true)
-  void createNewAnimal() {
+  void createNewProduct() {
 
-    AnimalDto animal = new AnimalDto();
-    animal.setName("Dog");
-    animal.setBasicInfo("Live");
-    animal.setNumberOfLegs(4);
+    ProductDto product = new ProductDto();
+    product.setTitle("Dog");
+    product.setBasicInfo("Live");
+    product.setPrice(BigDecimal.valueOf(1));
 
-    Response response = given().when().body(animal).contentType(MediaType.APPLICATION_JSON).post("/animals").then()
+    // product.setNumberOfLegs(4);
+
+    Response response = given().when().body(product).contentType(MediaType.APPLICATION_JSON).post("/products").then()
         .log().all().statusCode(201).header("Location", notNullValue()).extract().response();
 
     assertEquals(201, response.statusCode());
 
-    response = given().when().contentType(MediaType.APPLICATION_JSON).get("/animals").then().log().all().statusCode(200)
-        .extract().response();
+    response = given().when().contentType(MediaType.APPLICATION_JSON).get("/products").then().log().all()
+        .statusCode(200).extract().response();
 
-    PageResultDTO<AnimalDto> animalsReturned = response.as(new TypeRef<>() {
+    PageResultDTO<ProductDto> productsReturned = response.as(new TypeRef<>() {
     });
-    assertEquals(1, animalsReturned.getTotalElements());
-    AnimalDto created = animalsReturned.getStream().get(0);
+    assertEquals(1, productsReturned.getTotalElements());
+    ProductDto created = productsReturned.getStream().get(0);
     assertNotNull(created);
-    assertEquals(animal.getName(), created.getName());
+    assertEquals(product.getTitle(), created.getTitle());
   }
 
   @Test
-  @WithDBData(value = "data/animal.xls", deleteBeforeInsert = true)
+  @WithDBData(value = "data/product.xls", deleteBeforeInsert = true)
```

Comment:
You also have to change the name of the file in `src/test/resources/data`. Also change the content, so that it does match with the test cases